src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,12 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import { rivetize } from '../Rivet';
+
+const Grid : React.SFC<React.HTMLAttributes<HTMLDivElement>> = ({ children, className, ...attrs }) => (
+    <div className={classNames('rvt-grid', className)} {...attrs}>
+        {children}
+    </div>
+);
+Grid.displayName = 'Grid';
+
+export default rivetize(Grid);
```

Comment:
If you, like me, wondered why `<div class="rvt-grid">` wasn't just added to `Container` and why `Grid` needs its own component, it's (presumably) because you can nest `Grid` components.

Reply:
I added an alias named `Row` to the `Grid` component because I also found it a bit confusing, but I was attempting to align the components with the Rivet documentation.

Reply:
Thanks, that sounds like a helpful addition. I was curious, so I went over to the Rivet docs to see how the grid class was intended to be used in cases other than as the direct child of a div with the container class and saw that use case.